tests/integration/suite/test_notifier.py

```diff
@@ -0,0 +1,62 @@
+from kubernetes.client import CustomObjectsApi
+from .common import random_str
+
+group = 'management.cattle.io'
+version = 'v3'
+namespace = 'local'
+plural = 'notifiers'
```

Comment:
This set of things should be held in a dict, preferably inside of `verifySmtpPassword`, as they are specific to that function and having them global can cause potential issues in the future.